src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2471,7 +2471,7 @@ End Class
 "<Extension> Function IEnumerable(Of 'a).ToArray() As 'a()
 
 Anonymous Types:
-    'a is New With { .x As Integer }")
+    'a is New With { Key .x As Integer }")
```

Comment:
This looks like a breaking change in behavior of the API, defaults are changed.

---
In reply to: [82080448](https://github.com/dotnet/roslyn/pull/14298#discussion_r82080448) [](ancestors = 82080448)